styles/layout/_sidebar.scss

```diff
@@ -0,0 +1,61 @@
+@use "../abstract/variables" as *;
+@use "../abstract/mixin" as *;
+
+.sidebar {
+  background-color: $sidebar-background;
+  width: $sidebar-width;
+  height: $sidebar-height;
+  top: 0;
+  left: 0;
+  position: fixed;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  & > .__upper-sidebar {
+    @include flex-horizontal-center;
+    @include flex-vertical-center(0.5rem);
+    margin-top: 10.81rem;
+    padding: 0;
+    list-style-type: none;
+  }
+  & > .__upper-sidebar li {
+    display: flex;
+    width: 3.875rem;
+    height: 2.4375rem;
+    padding: 0.125rem 0.375rem 0.125rem 0.3125rem;
+    justify-content: center;
+    align-items: center;
+    transition: all 0.2s ease-in-out;
+    border: 1px solid transparent;
+    border-radius: 0.5rem;
+    cursor: pointer;
+  }
+  & > .__upper-sidebar li:hover {
+    border: 1px solid $primary;
+    background: $sidebar-icon-bg;
+    & > svg,
+    & > svg path {
+      fill: $primary;
+    }
+  }
+  & > .__lower-sidebar {
+    @include flex-vertical-center(1rem);
+    padding: 0;
```

Comment:
this should all be nested

see this all over the app, things shouldnt be repeated